gramps/plugins/export/exportcsv.py

```diff
@@ -317,9 +251,9 @@ def write_csv(self, *items):
     def export_data(self):
         self.dirname = os.path.dirname (self.filename)
         try:
-            self.g = open(self.filename,"w")
-            self.fp = open(self.filename, "wb")
-            self.g = UnicodeWriter(self.fp)
+            self.fp = open(self.filename, "w", encoding='utf_8_sig',
```

Comment:
How else can a program automatically detect that the input is utf-8?  (I am just trying to get Gramps to accept its own export, which someone made utf-8 some time ago.)